include/wx/dcsvg.h

```diff
@@ -133,125 +146,124 @@ class WXDLLIMPEXP_CORE wxSVGFileDCImpl : public wxDCImpl
-                            const wxColour& WXUNUSED(col),
-                            wxFloodFillStyle WXUNUSED(style) = wxFLOOD_SURFACE) wxOVERRIDE
-   {
-       wxFAIL_MSG(wxT("wxSVGFILEDC::DoFloodFill Call not implemented"));
-       return false;
-   }
+    virtual bool DoFloodFill(wxCoord WXUNUSED(x), wxCoord WXUNUSED(y),
+                             const wxColour& WXUNUSED(col),
+                             wxFloodFillStyle WXUNUSED(style) = wxFLOOD_SURFACE) wxOVERRIDE
+    {
+        wxFAIL_MSG(wxT("wxSVGFILEDC::DoFloodFill Call not implemented"));
+        return false;
+    }
 
-   virtual void DoGetSize(int * x, int *y) const wxOVERRIDE
-   {
-       if ( x )
-           *x = m_width;
-       if ( y )
-           *y = m_height;
-   }
+    virtual void DoGetSize(int * x, int *y) const wxOVERRIDE
+    {
+        if ( x )
+            *x = m_width;
+        if ( y )
+            *y = m_height;
+    }
 
-   virtual void DoGetTextExtent(const wxString& string, wxCoord *w, wxCoord *h,
-                                wxCoord *descent = NULL,
-                                wxCoord *externalLeading = NULL,
-                                const wxFont *font = NULL) const wxOVERRIDE;
+    virtual void DoGetTextExtent(const wxString& string, wxCoord *w, wxCoord *h,
+                                 wxCoord *descent = NULL,
+                                 wxCoord *externalLeading = NULL,
+                                 const wxFont *font = NULL) const wxOVERRIDE;
 
-   virtual void DoSetDeviceClippingRegion(const wxRegion& region) wxOVERRIDE
-   {
+    virtual void DoSetDeviceClippingRegion(const wxRegion& region) wxOVERRIDE
+    {
         DoSetClippingRegion(region.GetBox().x, region.GetBox().y,
                             region.GetBox().width, region.GetBox().height);
-   }
+    }
 
-   virtual void DoSetClippingRegion(int x,  int y, int width, int height) wxOVERRIDE;
+    virtual void DoSetClippingRegion(int x, int y, int width, int height) wxOVERRIDE;
 
-   virtual void DoGetSizeMM( int *width, int *height ) const wxOVERRIDE;
+    virtual void DoGetSizeMM(int *width, int *height) const wxOVERRIDE;
 
-   virtual wxSize GetPPI() const wxOVERRIDE;
+    virtual wxSize GetPPI() const wxOVERRIDE;
 
-   void Init (const wxString &filename, int width, int height,
+    void Init(const wxString &filename, int width, int height,
               double dpi, const wxString &title);
 
-   void write( const wxString &s );
+    void write(const wxString &s);
 
 private:
-   // If m_graphics_changed is true, close the current <g> element and start a
-   // new one for the last pen/brush change.
-   void NewGraphicsIfNeeded();
-
-   // Open a new graphics group setting up all the attributes according to
-   // their current values in wxDC.
-   void DoStartNewGraphics();
-
-   wxString            m_filename;
-   int                 m_sub_images; // number of png format images we have
```

Comment:
I think it must have been supposed to be static... not sure at all if it's worth having it, but I guess remembering the index of the last file written could save us quite a few iterations of the loop finding the next available file name. I.e. if you're saving 1000 SVGs (which is a lot but not really excessive), this could save half a million of `wxFileExists()`...

Reply:
The counter is still static (two actually), but it is now declared inside the function implementation.